src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -779,7 +779,28 @@ public IEnumerable<Element> GetJoinedElements()
                 .ToList();
         }
 
-
+        /// <summary>
+        /// Switch the order of how two elements are geometry is joined. 
+        /// Note that changing this in 3D will also affect your 2D views.
+        /// </summary>
+        /// <param name="cuttingElement">The element that should be cutting the other element</param>
+        /// <param name="otherElement">The other element that are being cut by the cuttingElement</param>
+        /// <returns>Joined elements</returns>
+        public static IEnumerable<Element> SwitchGeometryJoinOrder(Element cuttingElement, Element otherElement)
+        {
+            if (!JoinGeometryUtils.AreElementsJoined(Document, cuttingElement.InternalElement, otherElement.InternalElement))
+            {
+                return new List<Element>();
```

Comment:
feel like here we should throw since `switching join order` on something that is not already joined doesn't have the same result as when they are.
`throw new InvalidOperationException("Geometry join order can only be switched when the elements are already joined."); but obvs adding text to `.resx`